packages/flutter_test/lib/src/finders.dart

```diff
@@ -23,18 +23,42 @@ const CommonFinders find = CommonFinders._();
 class CommonFinders {
   const CommonFinders._();
 
-  /// Finds [Text] and [EditableText] widgets containing string equal to the
-  /// `text` argument.
+  /// Finds [Text], [EditableText], and optionally [RichText] widgets
+  /// containing string equal to the [text] argument.
+  ///
+  /// If [skipRichText] is disabled, [RichText] widgets (and therefore also
```

Comment:
here and elsewhere: method prams like skipRichText should be surrounded with `` instead of [].

Reply:
Thanks! I am wondering, is there a particular reason for this in dartdoc? In my IDE, I can use code navigation on parameters (so `skipRichText` correctly links to the parameter).

Reply:
I'd also love to know the reason behind this.